app.py

```diff
@@ -132,8 +132,9 @@ def graph_refresh_solver() -> None:
                 indexes=indexes,
                 output=_SOLVER_OUTPUT,
                 subgraph_check_api=_SUBGRAPH_CHECK_API,
+                transitive=False,
```

Comment:
why not?!

Reply:
This will cause the parallel execution of dependency (sub)graphs as shown in yesterday's demo.